app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -125,6 +142,10 @@ public void onReceive(Context ctx, Intent intent) {
         updateCurrentBgInfo();
     }
 
+    public static Context getContext() {
```

Comment:
we should not do this, its going to cause the app to crash if home gets oom cleaned up